src/gordon_gcp/clients/auth.py

```diff
@@ -173,12 +174,14 @@ def _setup_request_body(self):
             :exc:`.GCPHTTPError`: if any exception occurred.
         """
         url, headers, body = self._setup_token_request()
-
-        logging.debug(_utils.REQ_LOG_FMT.format(method='POST', url=url))
+        request_id = uuid.uuid4()
+        logging.debug(_utils.REQ_LOG_FMT.format(
+            request_id=request_id, method='POST', url=url, kwargs=None))
         async with self._session.post(url, headers=headers, data=body) as resp:
             log_kw = {
+                'request_id': request_id,
                 'method': 'POST',
-                'url': url,
+                'url': resp.url,
```

Comment:
I'm not sure what the difference is?

Reply:
`url` is the URL that gets passed in, and AFAICT [resp.url](https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientResponse.url) is the [effective URL](https://docs.aiohttp.org/en/stable/client_quickstart.html#passing-parameters-in-urls) that gets called 

I'm not 100% because then I'm sure what [real_url](https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientResponse.real_url) is, but I think `resp.url` is better than just `url` in any case.